plugin/pki/util_test.go

```diff
@@ -0,0 +1,17 @@
+package pki
+
+import (
+	"testing"
+)
+
+func TestSHA1SUM(t *testing.T) {
+	// Known SHA1SUM value of "hello"
+	// echo -n "hello" | sha1sum
+	SHA1SUMstringValue := "aaf4c61ddcc5e8a2dabede0f3b482cd9aea9434d"
+
+	s := "hello"
+	SHA1SUMvalue := sha1sum(s)
+	if SHA1SUMstringValue != SHA1SUMvalue {
+		t.Fatalf("%s function", GetFunctionName(sha1sum))
```

Comment:
Why a function to extract the name of the sha1 function is rrequired here? Shouldnt a fixed value be better?
Even better shouldn't a meaningful error message be returned here?

Reply:
I was trying to denote which function is the one failing. But yes, I should have put a better error message